pyocd/probe/picoprobe.py

```diff
@@ -104,8 +104,13 @@ def close(self):
     @classmethod
     def enumerate_picoprobes(cls, uid=None) -> List["PicoLink"]:
         """! @brief Find and return all Picoprobes """
-        # Use a custom matcher to make sure the probe is a Picoprobe and accessible.
-        return [PicoLink(probe) for probe in core.find(find_all=True, custom_match=FindPicoprobe(uid))]
+        try:
+            # Use a custom matcher to make sure the probe is a Picoprobe and accessible.
+            return [PicoLink(probe) for probe in core.find(find_all=True, custom_match=FindPicoprobe(uid))]
+        except core.NoBackendError:
+            LOG.debug(" No usb backend found")
```

Comment:
Change to make use of `pyocd.probe.common.show_no_libusb_warning()`, and update the message in that function.

Reply:
I had noticed that, and was wondering whether it was a better choice.
In the end I decided not change an unrelated file, but it's a better solution so I'll go for it.

Would you suggest adding picoprobe explicitly or just changing it to a generic message such as "Libusb is not installed: some probe types cannot be supported" and adding PP to the documentation?

Reply:
Ok, I'll make this last change!